crypto/crypto-derive/src/hasher.rs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+/// Converts a camel-case string to snake-case
+pub fn camel_to_snake(text: &str) -> String {
```

Comment:
Why do we require this?

Reply:
Not a requirement at all, we just happened to discuss it offline :)